pkg/services/contexthandler/contexthandler.go

```diff
@@ -141,6 +142,17 @@ func (h *ContextHandler) Middleware(next http.Handler) http.Handler {
 			reqContext.Resp.Before(h.addIDHeaderEndOfRequestFunc(reqContext.SignedInUser))
 		}
 
+		// Set open feature evaluation context with namespace
+		ns := "default"
```

Comment:
In case if request is not authenticated, `id` is nil, so I think I need to fallback to some value. For now I put `"default"` as a namespace value.